cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/migration/MigrationWhenAFaultHappensExample.java

```diff
@@ -0,0 +1,315 @@
+package org.cloudsimplus.migration;
+
```

Comment:
You missed this file.
The methods and attributes that aren't constants must be non-static.
Rename the file to MigrationWhenHostFaultHappensExample.java